Resources/Prototypes/Nyanotrasen/Entities/Clothing/Head/hats.yml

```diff
@@ -8,6 +8,11 @@
     sprite: Nyanotrasen/Clothing/Head/Hats/mailman.rsi
   - type: Clothing
     sprite: Nyanotrasen/Clothing/Head/Hats/mailman.rsi
+- type: Tag # DeltaV
+  tags: # DeltV
+  - ClothMade # DeltaV
+  - WhitelistChameleon # DeltaV
+  - CatWearable # Frontier
```

Comment:
Its a comp not a tag so we dont need to fuck with tags
CatWearable

Reply:
ohhhh my bad